scripts/dockerfiles/run.sh

```diff
@@ -85,6 +90,58 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    psp=$(find /var/vcap/jobs/*/bin -name post-start)
+
+    if [ "X$psp" != X ] ; then
```

Comment:
Do we need the check here, or just always write out the job and have the post-start script do nothing?